.github/actions/setup-alpine/action.yml

```diff
@@ -17,6 +17,7 @@ runs:
           automake \
           bash \
           build-base \
+          clang \
```

Comment:
commit message:
"Note: Fedora 40 doesn't ship shared the shared sanitizer library, while"

one shared too many